cones-challenge/main.py

```diff
@@ -86,7 +109,7 @@ def frameNorm(w, h, bbox):
         self.last_cones_distances = []
         for detection in self.last_detections:
             # ['cone', 0.92236328125, [0.42129743099212646, -0.0010452494025230408, 0.4836755692958832, 0.1296510100364685]]
-            w, h = 640, 400
+            w, h = 1280, 720  #640, 400
```

Comment:
Could be defined outside of the for-cycle and auto loaded from a depth frame?

Reply:
yes, this is an old hack ... I still do not like the proper scaling used in NN